examples/hn.svelte.dev/vite.config.js

```diff
@@ -1,9 +1,18 @@
-import { resolve } from 'path';
+// Consult https://vitejs.dev/config/ to learn about these options
+import { join, resolve } from 'path';
+import { readFileSync } from 'fs';
+import { cwd } from 'process';
 
+const pkg = JSON.parse(readFileSync(join(cwd(), 'package.json')));
+
+/** @type {import('vite').UserConfig} */
 export default {
 	resolve: {
 		alias: {
 			$components: resolve('src/components')
 		}
+	},
+	ssr: {
+		noExternal: [...Object.keys(pkg.dependencies || {})]
```

Comment:
Object.keys is already an array, so no need to spread like with the ones that return iterators.

Reply:
I had it as `noExternal: Object.keys(pkg.dependencies || {})` at first, but then I remembered that when we still had Snowpack we did something similar:

https://github.com/sveltejs/kit/blob/55ccb31551216bd4b2060fdc85a5e17e41ac4aae/packages/snowpack-config/snowpack.config.cjs#L10

Then again that wasn't really user-facing and it was concatenating two arrays, so I guess I'll just use `Object.keys` directly.